gpkit/tests/t_nomials.py

```diff
@@ -237,21 +243,25 @@ def test_init(self):
         x = Monomial('x')
         y = Monomial('y')
         ms = [Monomial({'x': 1, 'y': 2}, 3.14),
-              Monomial('y', 0.5),
+              0.5*Monomial('y'),
               Monomial({'x': 3, 'y': 1}, 6),
-              Monomial({}, 2)]
+              Monomial(2)]
         exps, cs = [], []
         for m in ms:
             cs += m.cs.tolist()
             exps += m.exps
-        p = Posynomial(exps, cs)
+        hmap = NomialMap(zip(exps, cs))
+        hmap.set_units(None)
```

Comment:
there is no default right now, though we could set `Nomial.Map.units = None`. I did this for debug purposes as much as anything else, but it might be confusing to users.